### ISSUE TITLE
pressing start menu on the second monitors opens it on primary monitor

### ISSUE THREAD

**Issue by KoleckOLP** [valinet/ExplorerPatcher#1529]
![image](https://github.com/valinet/ExplorerPatcher/assets/22729896/c8e7c592-2671-4955-a136-e8e6dc96252a)

primary screen is 2560x1440 
secondary screen is 1920x1080

pressing the start button opens it on the primary screen in the 1920x1080 location

settings export (zipped):
[ExplorerPatcher_22621.1555.55.1.reg.zip](https://github.com/valinet/ExplorerPatcher/files/11666845/ExplorerPatcher_22621.1555.55.1.reg.zip)

**Comment by pyrates999** [comment]
You have a preview update installed: https://support.microsoft.com/en-us/topic/may-24-2023-kb5026446-os-build-22621-1778-preview-3c547100-7a73-4ae6-bb7d-ebd02e87dc04

Preview updates and insider builds are not supported because symbols cannot be downloaded for them: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

Uninstall the preview update, reboot, and then check to see if symbols have been downloaded.

**Comment by KoleckOLP** [comment]
Hello thank you for that info I didn't know microsoft provides symbols,

I unisntalled that update and my Windows 11 is not 22621.1702 which is still not the correct version.

**Comment by KoleckOLP** [comment]
Uninstalling all updates that I can unintall only gets me to Windows 11 22621.1636

**Comment by pyrates999** [comment]
Please show a screen shot from winver.  Google doesn't show much for the build 22621.1636 that you provided.

**Comment by pyrates999** [comment]
22621.1635 is the april preview update: https://support.microsoft.com/en-au/topic/april-25-2023-kb5025305-os-build-22621-1635-preview-2482528b-7e8e-4179-b255-808f484b9c0c

So uninstall KB5025305 and reboot.

You gotta keep checking until you have no preview updates installed.

**Comment by Hidekikun** [comment]
Same issue, don't appear to have any preview updates. Per searching. After it said it downloaded symbols clicked to restart it now it appears on the wrong monitor and refuses to move to any other.

![image](https://github.com/valinet/ExplorerPatcher/assets/17399326/ff1ef479-0fdd-42f7-9d58-d9f993479ee3)

Anything else I can check?

**Comment by pyrates999** [comment]
After symbols have been downloaded, in the explorer patcher properties application, click on restart file explorer:

https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Comment by pyrates999** [comment]
Did you get a notification that symbols were downloaded though?

**Comment by Hidekikun** [comment]
> Did you get a notification that symbols were downloaded though?

Yes correct, only works w/o the updates so I have them disbaled for now.

**Comment by reeeno** [comment]
I have the same issue. ExplorerPatcher is on version 22621.2861.62.2, and running on Windows 11 Pro version 22H2 Build 22621.2861. Clicking the start menu button on second/third monitors opens the start menu on the primary monitor at the pixel respective to second/third monitor's resolution.
